### PR TITLE
Implement command to freeze ANJ bonding curve

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

**Review by sohkai — APPROVED** [review]
LGTM!

File: src/models/apps/BondingCurveFreezer.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    const { governor, kernel: kernelAddress, fundraising } = this.config
+    logger.info('Building EVM script to revoke ANJ bonding curve permissions...')
+
+    const Kernel = await this.environment.getArtifact('Kernel', '@aragon/os')
+    const kernel = await Kernel.at(kernelAddress)
+    const acl =  await kernel.acl()
+
+    const permissions = ['OPEN_BUY_ORDER_ROLE', 'OPEN_SELL_ORDER_ROLE']
+    const callsScript = permissions.map(role => {
+      const script = this.encoder.encodeRevokePermission(ANY_ENTITY, fundraising, sha3(role))
```
Comment (0xGabi) [inline src/models/apps/BondingCurveFreezer.js:26]: To double-check here if we assign `ANY_ENTITY` then the ACL `revokePermission` method sets `NO_PERMISSION` (bytes32(0)) for every entity

**Review by 0xGabi — APPROVED** [review]
Looks good to me 👍

File: src/models/apps/BondingCurveFreezer.js (hunk 1 above)
Comment (facuspagnuolo) [reply src/models/apps/BondingCurveFreezer.js:26]: `NO_PERMISSION` is actually for the params, this call will simply remove the permission from `ANY_ENTITY`

**Pull request merged**